test/AirDrop.test.js

```diff
@@ -0,0 +1,327 @@
+                expect(await instanceToken.balanceOf(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+                await instanceToken.approve(instanceAirDrop.address, ether('5'));
+                await expectRevert(instanceAirDrop.depositTokens(ether('5'), { from: acc2 }), "Error : caller is not the owner");
+                expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('100'));
+            });
+        });
+
+        describe("depositTokens - done", async () => {
+            it("call 'depositToken' function - done", async () => {
+                expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('100'));
+                expect(await instanceToken.balanceOf(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+                await instanceToken.approve(instanceAirDrop.address, ether('5'));
+                let tx = await instanceAirDrop.depositTokens(ether('5'));
+                expect(await instanceToken.balanceOf(instanceAirDrop.address)).to.be.bignumber.equal(ether('5'));
+                expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('95'));
+                await expectEvent(tx, "DepositTokens", { amount: ether('5') });
+            });
+        });
+    });
+
+    describe("depositEther", async () => {
+        describe("depositEther - false", async () => {
+            it("call 'depositEther' function - false (amount = 0)", async () => {
+                expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+                await expectRevert(instanceAirDrop.depositEther({ from: owner, value: web3.utils.toWei("0", "ether") }), "Error : 'Amount' , equal to 0");
+                expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+            });
+
+            it("call 'depositEther'function - false (caller not owner)", async () => {
+                expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+                await expectRevert(instanceAirDrop.depositEther({ from: acc2, value: web3.utils.toWei("1", "ether") }), "Error : caller is not the owner");
+                expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+            });
+        });
+
+        describe("depositEther - done", async () => {
+            it("call 'depositEther' function - done ", async () => {
+                expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('0'));
+                let tx = await instanceAirDrop.depositEther({ from: owner, value: web3.utils.toWei("10", "ether") });
+                expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('10'));
+                await expectEvent(tx, "DepositEther", { amount: ether('10') });
```

Comment:
also check how many ether was transfer from sender. you only check  if contract balance was change.

user balancer from openzeppelin instand web3.eth.getBalance